docs/ui/components/Snippet/SnippetExpandOverlay.tsx

```diff
@@ -6,8 +6,8 @@ type Props = {
 
 export function SnippetExpandOverlay({ onClick }: Props) {
   return (
-    <div className="absolute bottom-0 left-0 flex w-full bg-gradient-to-b from-transparent to-default p-6">
-      <Button theme="secondary" onClick={onClick} className="mx-auto">
+    <div className="sticky bottom-0 left-0 flex w-full bg-gradient-to-b from-transparent to-default p-6">
```

Comment:
Changing it to sticky makes the part behind the Show more button invisible, rather than the effect we have currently on prod. Is this expected? If yes, is it possible to make it more silent?

<img width="1112" height="1162" alt="CleanShot 2025-08-01 at 16 52 16@2x" src="https://github.com/user-attachments/assets/6aa5f205-f7de-4550-b3c2-63a2a0947c3e" />

<img width="1098" height="1160" alt="CleanShot 2025-08-01 at 16 52 19@2x" src="https://github.com/user-attachments/assets/107399e3-a390-49d0-9723-9a335841a35e" />